### DISCUSSION TITLE
Como vai ser o registro de saldo do cliente?

### DISCUSSION THREAD

**Discussion by lucassl2020** [EuricoDNJR/SG-VILLA-DOLCE#74] [Q&A] [answered]
O product owner quer que o cliente resgate um açaí ao completar saldo de 150 reais (10 pontos). Ou, prefere dar um desconto na próxima compra do cliente ao atingir saldo de 150?

**Comment by EuricoDNJR** [comment, marked as answer]
O product owner disse que prefere dar um desconto de **15 reais** na próxima compra de **Açaí** do cliente.
